tools/update_runnablejobs.py

```diff
@@ -62,19 +63,25 @@ def update_runnableapi():
         else:
             print "It's going to update your runnable jobs data."
             data = query_the_runnablejobs(new_timestamp, task_id)
-            new_jobs = add_jobs_to_jobpriority(new_data=data,
-                                               priority=1,
-                                               timeout=0,
-                                               set_expired=True)
+            # We're calling with set_expired to True since we expect the new
+            # runabble file might have new jobs
+            # XXX: How do we know that in this case we're going to have new jobs instead of
+            #      the other case below? Does the timestamp in meta that runnable apis give us
+            #      mean that it actually has new data?
```

Comment:
My apologies. I didn't realize that I had included the commented out code and the sys.exit()